app/components/infiniteScroll.tsx

```diff
@@ -0,0 +1,60 @@
+import {ReactNode, useState, useRef, useCallback} from 'react'
+
+type Props = {
+  children: ReactNode[]
+  perPage: number
+  fetchMore: (page: number, perPage: number) => any[]
+  loadingRenderer: (loading: boolean) => ReactNode
+  [k: string]: any
```

Comment:
is there some non-any type we could use, please? e.g. `JSX.IntrinsicElements['div']` or `React.HTMLAttributes<HTMLDivElement>`?

Reply:
nice!